Modules/_sqlite/cache.c

```diff
@@ -113,7 +117,8 @@ void pysqlite_cache_dealloc(pysqlite_Cache* self)
     Py_DECREF(tp);
 }
 
-PyObject* pysqlite_cache_get(pysqlite_Cache* self, PyObject* key)
+PyObject *
+pysqlite_cache_get(pysqlite_Cache *self, PyObject *key)
```

Comment:
I really don't like these unrelated cosmetic changes. The scope of this PR is adding missing `static` keywords and it's OK to follow PEP 7 while adding them. However, changes like these are out of scope of this PR.

Reply:
> The scope of this PR is adding missing `static` keywords and it's OK to follow PEP 7 while adding them.

That's my impression from other PR's that has been reviewed by other core devs; embedding PEP 7 style changes seems to be the default, AFAICT.

> However, changes like these are out of scope of this PR.

Isn't that a contradiction of what you just said? :)

Reply:
> That's my impression from other PR's that has been reviewed by other core devs; embedding PEP 7 style changes seems to be the default, AFAICT.

That's true as long as you've already touched the same lines as part of the original task you were working on. It doesn't mean you can randomly apply style changes all over the place.

> Isn't that a contradiction of what you just said? :)

No. Most of the changes here are purely cosmetic (i.e. they already have the `static` keyword in-place) It's OK to follow PEP 7 while adding *missing* `static` keywords.

Reply:
To be clear, the following diff is OK:

```diff
- int foo(...)
+ static int
+ foo(...) 
{

}

void bar(...)
{

}

static void baz(...)
{

}
```

But this is not OK:

```diff
- int foo(...)
+ static int
+ foo(...) 
{

}

- void bar(...)
+ void
+ bar(...)
{

}

- static void baz(...)
+ static void
+ baz(...)
{

}
```

Reply:
BTW 1, Would you accept a PR that cleans up the remaining ~40 argument specs wrt. PEP 7?

BTW 2, neat with inline syntax highlighting in the GH markup. I like that. Thanks.